examples/virtio/README.md

```diff
@@ -56,6 +56,25 @@ Welcome to Buildroot
 buildroot login:
 ```
 
-You will notice that we have two Buildroot login prompts, initially all input is defaulted
-to the guest one (in red). To switch to input into the other guest (green), type in `@2`.
-The `@` symbol is used to switch between clients, in this case the green guest is client 2.
+Initially all input is defaulted
+to guest 1 in green. To switch to input into the other guest (red), type in `@2`.
+The `@` symbol is used to switch between clients of the serial system, in this case the red guest is client 2.
+
+### virtIO block
+
+Guest 1 and guest 2 also doubles as a client in the block system that talks virtIO to guest 3 that acts as driver with passthrough access to the block device. The requests from both clients are multiplexed through the additional block virtualiser component.
```

Comment:
Make sure these lines don't exceed 80-100 lines like the rest of the README since some people will read this in a terminal